packages/third-parties/src/ThirdPartyScriptEmbed.tsx

```diff
@@ -15,6 +17,16 @@ export default function ThirdPartyScriptEmbed({
   children,
   dataNtpc = '',
 }: ScriptEmbed) {
+  useEffect(() => {
+    // Useful for feature detection and measurement
+    performance.mark('mark_use_counter', {
+      detail: {
+        feature: 'next-third-parties',
+        type: dataNtpc,
```

Comment:
Instead of using 'type', can we append the `dataNtpc` to the main feature name if it exists? e.g. `next-third-parties-${dataNtpc}`